go/enclave/evm/ethchainadapter/eth_chainadapter.go

```diff
@@ -56,15 +56,12 @@ func (e *EthChainAdapter) CurrentBlock() *gethtypes.Header {
 	if currentBatchSeqNo == nil {
 		return nil
 	}
-	ctx, cancelCtx := context.WithTimeout(context.Background(), e.config.RPCTimeout)
-	defer cancelCtx()
-
-	currentBatch, err := e.storage.FetchBatchHeaderBySeqNo(ctx, currentBatchSeqNo.Uint64())
+	currentBatch, err := e.storage.FetchBatchHeaderBySeqNo(context.Background(), currentBatchSeqNo.Uint64())
```

Comment:
given this function doesn't return an error, it's safest to not use timeouts.

I've seen it fail with "context cancelled" which ended up in a panic